### ISSUE TITLE
linter skips selector with only @extends

### ISSUE THREAD

**Issue by bencripps** [postcss/postcss-bem-linter#118]
With the following CSS, the `hasNoDeclarations` function in get-selectors is returning true, which means we are not linting the selector name. (We're running our code through a couple plugins, mentioned below, but it looks like those are behaving correctly). 

```sass
/** @define Foo */

.namespace-Component {
    @extend .myMixin;
}

```
I would expect this to provide a warning, but since the hasNoDeclarations function returns false, we early return before linting the selector name. I see the following comment:

```sass
  // Skip validation on rules with no declarations
  // as these don't exist after rules have been unwrapped
```
so I assume this is the expected behavior, but I wonder if we should add a clause for extends and other `atrules`?

We're using the following versions:

```
postcss: 5.2.17
postcss-bem-linter: 2.7.0
stylelint-selector-bem-pattern: 1.0.0
stylelint-config-suitcss: 7.0.0.
```

**Comment by simonsmith** [comment]
In your example what CSS would you expect the linter to be checking?

**Comment by bencripps** [comment]
I'd prefer that the linter not skip over this selector so we can verify we the class name matches our naming conventions (defined by `stylelint-selector-bem-pattern`). 

In this case `.namespace-Component` does not match the comment at the top, so I would expect to see an error in the lint output.

**Comment by simonsmith** [comment]
Makes sense. Sounds like what you suggested is the sensible route and we don't assume these selectors are empty.

**Comment by simonsmith** [comment]
Fixed in #119
